core/scripts/cli/cli_entry.sh

```diff
@@ -8,7 +8,7 @@ set -e
 # $(npm bin)/uma --network mainnet_ledger
 
 # Platform independent way of evaluating symlinks to find the real location of the script.
-SCRIPT_FILE=$(/usr/bin/env python2 -c "import os; print(os.path.realpath(\"$0\"))")
+SCRIPT_FILE=$(/usr/bin/env python -c "import os; print(os.path.realpath(\"$0\"))")
```

Comment:
why'd you change this?

Reply:
It doesn't work on my machine with `python2`.